projects/fal/src/fal/toolkit/file/providers/fal.py

```diff
@@ -672,7 +788,7 @@ def save_file(
             multipart = os.path.getsize(file_path) > threshold
 
         if multipart:
-            url = self._save_multipart(
+            url = MultipartUpload.save_file(
```

Comment:
since now `.save` handles this, you could just forward all the multipart-related args to it and leave the decision to `save`

Reply:
you mean to self.save? It only handles bytes though, but here we need to handle file, so that we don't read it all into memory.